clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -266,7 +266,14 @@ public void add(TopicPartition topicPartition, PartitionData data) {
             }
         }
 
+        /**
+         * Build a FetchRequestData for the provided partitions
+         * @throws IllegalStateException if it has already been called
+         */
         public FetchRequestData build() {
+            if (next == null) {
+                throw new IllegalStateException("build() has already been called.");
```

Comment:
This is not a test change?

Reply:
I've reverted that change, I'll address this issue in another PR.